src/guidellm/scheduler/strategy.py

```diff
@@ -1,493 +1,700 @@
+        :return: String identifier for throughput strategy
         """
-        return "async"
+        return "throughput"
 
     @property
-    def queued_requests_limit(self) -> int:
+    def processes_limit(self) -> int | None:
         """
-        The maximum number of queued requests for the scheduling strategy.
-        It determines how many requests can be queued at one time
-        for the scheduling strategy and must be implemented by subclasses.
+        Get maximum number of worker processes for throughput scheduling.
 
-        :return: The processing requests limit to ensure that there are enough
-            requests even for the worst case scenario where the max concurrent
-            requests are pulled at once for processing.
+        :return: The max_concurrency value if set, otherwise None for unlimited
         """
-        return self.processing_requests_limit
+        return self.max_concurrency
 
     @property
-    def processing_requests_limit(self) -> int:
+    def requests_limit(self) -> int | None:
         """
-        The maximum number of processing requests for the scheduling strategy.
-        It determines how many requests can be processed at one time
-        for the scheduling strategy and must be implemented by subclasses.
+        Get maximum number of concurrent requests for throughput scheduling.
 
-        :return: {self.max_concurrency} for the throughput scheduling strategy to limit
-            the processing requests to the maximum concurrency.
-            If max_concurrency is None, then the default processing requests limit
-            will be used.
+        :return: The max_concurrency value if set, otherwise None for unlimited
         """
-        return self.max_concurrency or super().processing_requests_limit
+        return self.max_concurrency
 
-    def request_times(self) -> Generator[float, None, None]:
+    def create_request_timings(
+        self, local_rank: int, local_world_size: int, local_max_concurrency: int
+    ) -> ScheduledRequestTimings:
         """
-        A generator that yields the start time.time() so requests are sent
-        immediately, while scheduling as many asynchronously as possible.
+        Create timing implementation for throughput request scheduling.
 
-        :return: A generator that yields the start time.time()
-            for immediate request scheduling.
+        :param local_rank: The rank of the worker process
+        :param local_world_size: Total number of worker processes
+        :param local_max_concurrency: Maximum concurrent requests for the worker
+        :return: NoDelayRequestTimings instance for immediate request scheduling
         """
-        start_time = time.time()
+        if self.startup_duration > 0:
+            # Vary offset by up to 5% of the startup duration for a bit of variance
+            offset = 0.05 * self.startup_duration * (local_rank / local_world_size)
```

Comment:
Do we want this hard-coded?

Reply:
Yeah, we'll need to rework this logic in the future, but there's no way to enable it currently, so I'd say let's revisit it after this goes out